tests/Aspire.Hosting.Tests/SlimTestProgramTests.cs

```diff
@@ -18,6 +19,7 @@ public SlimTestProgramTests(SlimTestProgramFixture slimTestProgramFixture)
     }
 
     [Fact]
+    [ActiveIssue("https://github.com/dotnet/aspire/issues/7923", typeof(PlatformDetection), nameof(PlatformDetection.IsRunningOnGithubActions), nameof(PlatformDetection.IsWindows))]
```

Comment:
I mentioned it in https://github.com/dotnet/aspire/issues/7923, but maybe these should just have `[RequiresDocker]` attributes?

Reply:
Why are these not failing on the rolling builds then?

Reply:
That's a good question. Are they running there?

Reply:
Yeah - 
<img width="1080" alt="Image" src="https://github.com/user-attachments/assets/e71e7a99-b25c-4aac-86f4-5c27d0e56609" />
https://dev.azure.com/dnceng-public/public/_build/results?buildId=965263&view=ms.vss-test-web.build-test-results-tab